Testscripts/Windows/CPU-Offline.ps1

```diff
@@ -28,6 +31,8 @@ function Main {
 
 	try {
 		$testResult = $resultFail
+		$isStorageWorkloadEnable = 0
```

Comment:
why assign this twice

Reply:
Fixed in 3873e9bd982c480954f2da56d376acfcd68592e0